packages/docusaurus/src/client/App.js

```diff
@@ -16,8 +17,29 @@ import PendingNavigation from './PendingNavigation';
 import './client-lifecycles-dispatcher';
 
 function App() {
+  const {stylesheets, scripts} = siteConfig;
   return (
     <DocusaurusContext.Provider value={{siteConfig}}>
+      {(stylesheets || scripts) && (
+        <Head>
+          {stylesheets &&
+            stylesheets.map(source =>
+              source.href ? (
+                <link rel="stylesheet" key={source.href} {...source} />
+              ) : (
+                <link rel="stylesheet" key={source} href={source} />
+              ),
+            )}
+          {scripts &&
```

Comment:
Do we inject scripts here in V1? Putting scripts here blocks page rendering which is bad, unless they use `async`. Maybe we should allow injecting before `</body>`, but that'll be more complicated.

Reply:
yep its done like this in v1